apps/condo/domains/organization/integrations/sbbol/sync/syncOrganization.spec.js

```diff
@@ -83,6 +84,35 @@ describe('syncOrganization from SBBOL', () => {
             expect(existedEmployee.isAccepted).toBeTruthy()
             expect(existedEmployee.role.canManageEmployees).toBeTruthy()
         })
+
+        it('should create new organization and create new user', async () => {
+            const { userData, organizationData, dvSenderFields } = MockSbbolResponses.getUserAndOrganizationInfo()
+            const adminContext = await keystone.createContext({ skipAccessControl: true })
+            const context = {
+                keystone,
+                context: adminContext,
+            }
+            const user = await syncUser({ context, userInfo: userData })
+            await syncOrganization({
+                context,
+                user,
+                userData,
+                dvSenderFields,
+                organizationInfo: organizationData,
+            })
+            const [ newOrganization ] = await OrganizationApi.getAll(adminContext, {
+                importId: organizationData.importId,
+                importRemoteSystem: organizationData.importRemoteSystem,
+            })
+            expect(newOrganization).toBeDefined()
+            const [ existedEmployee ] = await OrganizationEmployeeApi.getAll(adminContext, {
```

Comment:
why you don't check the ExternalIdentity here?

Reply:
Organization does not have ExternalIdentity. UserExternalIdentity only rely to the user